app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
```

Comment:
```suggestion
class CityController
```
In this case extending `Controller` doesn't make sense unless we're gonna use anything from it. You can safely remove it.